spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/hystrix/SleuthHystrixConcurrencyStrategy.java

```diff
@@ -13,6 +13,14 @@
 import com.netflix.hystrix.strategy.HystrixPlugins;
 import com.netflix.hystrix.strategy.concurrency.HystrixConcurrencyStrategy;
 
+/**
+ * A {@link HystrixConcurrencyStrategy} that wraps a {@link Callable} in a
```

Comment:
HystrixTraceCallable is private, so linking to it doesn't help much.

something like..

wrapped callables are logged in the current span, or start a new one.